File: EPS FrontEnd/EPS/src/app/models/Protocol/protocol-request.ts
import { ProtocolItem } from './protocol-item';

export class ProtocolRequest
{
  UserCode: string;
  /// <summary>
  /// Historia
  /// </summary>
  History: number;
  /// <summary>
  /// Listado de protocolos
  /// </summary>
  Protocols: ProtocolItem[];
}

File: EPS BackEnd/EPS.WebServices/EPS.Common.Models/Protocol/ProtocolRequest.cs
using System.Collections.Generic;

namespace EPS.Common.Models
{
    /// <summary>
    /// Parametro de insercion protocolo
    /// </summary>
    public class ProtocolRequest
    {
        public string UserCode { get; set; }
        /// <summary>
        /// Historia
        /// </summary>
        public int History { get; set; }
        /// <summary>
        /// Listado de protocolos
        /// </summary>
        public List<ProtocolItem> Protocols { get; set; }
    }
}
